moviepy/video/io/ffmpeg_tools.py

```diff
@@ -328,5 +328,8 @@ def ffplay_version():
     result = subprocess.run(cmd, capture_output=True, text=True, check=True)
     # Extract the version number from the first line of output
     full_version = result.stdout.splitlines()[0].split()[2]
+    #On Arch linux ffmpeg versions start with n, then the version number, this makes sure that no errors occur.
```

Comment:
I can't comment on the code side of this, but would ask you clean up the comment. Spelling of proper names and punctuation should be fixed (missing whitespace after # for consistency with other comments; use full stop or semi-colon to split the sentence in two for readability).